widgets/range-slider/__tests__/range-slider-test.js

```diff
@@ -80,86 +47,157 @@ describe('rangeSlider()', () => {
-      start: [-Infinity, Infinity],
-      step: 1,
-      templateProps: {
-        templates: {footer: '', header: ''},
-        templatesConfig: undefined,
-        transformData: undefined,
-        useCustomCompileOptions: {footer: false, header: false}
-      },
-      tooltips: true
-    };
-
-    expect(ReactDOM.render.calledTwice).toBe(true, 'ReactDOM.render called twice');
-    expect(autoHideContainer.calledOnce).toBe(true, 'autoHideContainer called once');
-    expect(headerFooter.calledOnce).toBe(true, 'headerFooter called once');
-    expect(ReactDOM.render.firstCall.args[0]).toEqualJSX(<Slider {...props} />);
-    expect(ReactDOM.render.firstCall.args[1]).toEqual(container);
-    expect(ReactDOM.render.secondCall.args[0]).toEqualJSX(<Slider {...props} />);
-    expect(ReactDOM.render.secondCall.args[1]).toEqual(container);
-  });
-
-  it('doesn\'t call the refinement functions if not refined', () => {
-    let state0 = helper.state;
-    widget.render({results, helper});
-    let state1 = helper.state;
-    expect(state1).toEqual(state0);
-    expect(helper.search.called).toBe(false, 'search never called');
-  });
-
-  it('calls the refinement functions if refined with min+1', () => {
-    let stats = results.disjunctiveFacets[0].stats;
-    let targetValue = Math.floor(stats.min) + 1;
-
-    let state0 = helper.state;
-    widget._refine(helper, stats, [targetValue, stats.max]);
-    let state1 = helper.state;
-
-    expect(helper.search.calledOnce).toBe(true, 'search called once');
-    expect(state1).toEqual(state0.addNumericRefinement('aNumAttr', '>=', targetValue));
-  });
-
-  it('calls the refinement functions if refined with max-1', () => {
-    let stats = results.disjunctiveFacets[0].stats;
-    let targetValue = Math.ceil(stats.max) - 1;
-
-    let state0 = helper.state;
-    widget._refine(helper, stats, [stats.min, targetValue]);
-    let state1 = helper.state;
-
-    expect(helper.search.calledOnce).toBe(true, 'search called once');
-    expect(state1).toEqual(state0.addNumericRefinement('aNumAttr', '<=', targetValue));
+  context('without result', () => {
+    beforeEach(() => {
+      results = {};
+    });
+
+    it('calls ReactDOM.render(<Slider props />, container)', () => {
+      widget.render({results, helper});
+
+      let props = {
+        cssClasses: {body: null, root: null},
+        onChange: () => {},
+        pips: true,
+        range: {max: 0, min: 0},
+        shouldAutoHideContainer: true,
+        start: [-Infinity, Infinity],
+        step: 1,
+        templateProps: {
+          templates: {footer: '', header: ''},
+          templatesConfig: undefined,
+          transformData: undefined,
+          useCustomCompileOptions: {footer: false, header: false}
+        },
+        tooltips: true
+      };
+
+      expect(ReactDOM.render.calledOnce).toBe(true, 'ReactDOM.render called twice');
+      expect(autoHideContainer.calledOnce).toBe(true, 'autoHideContainer called once');
```

Comment:
Isn't the purpose of the message is to indicate the error we detected? In this case, wouldn't it be like on the previous line?